### ISSUE TITLE
Click events activating randomly ("Ghost click")

### ISSUE THREAD

**Issue by snavas** [AR-js-org/AR.js#107]
**Do you want to request a *feature* or report a *bug*?**

I would like to report a bug

**What is the current behavior?**

A couple of weeks ago, I implemented click-based interaction using the legacy versions of AR.js and A-Frame used in the example described by the user @yannklein in [this old github issue](https://github.com/jeromeetienne/AR.js/issues/756). However, recently I don't know what it has changed, but attempting to use click based interactions with A-Frame 0.8.0 and AR.js 2.1.8 works on my laptop but it doesn't work on my android phone anymore.

I have worked around this issue by using A-Frame 0.9.2 and the latest version of AR.js or A-Frame 0.8.0 and the latest version of AR.js. However, although both versions work kind of ok-ish (despite raycasting being off as already reported in #40. Altough I have the feeling that raycasting is less off using A-Frame 0.8.0 than 0.9.2), I have noticed a behaviour I did not notice before, which is: **click events get activated randomly**, without me touching the screen.

**If the current behavior is a bug, please provide the steps to reproduce.**

I have put together two examples in codepen, [one using A-Frame 0.9.2](https://codepen.io/navasmedrano/pen/RwWxLGV) and [another one using A-Frame 0.8.0](https://codepen.io/navasmedrano/pen/mdepqXv). Both exhibit this ghost click behaviour where the clicks events get acctivated randomly without no one touching the screen.

**Please mention other relevant information such as the browser version, Operating System and Device Name**

Please note that this ghost click bug only happens on my android phone. It does not happen on my laptop. I tried already in two different android phones using different browsers (firefox and chrome). I can share more details if needed.

**Comment by kalwalt** [comment]
Hi @snavas can you share more details?  Does it happens with Pattern, Barcode NFT markers? Also I f you can share some piece of code, it would be super. Consider also that we did some changes to fix a bug https://github.com/AR-js-org/AR.js/pull/60

**Comment by snavas** [comment]
Hi @kalwalt,

Sorry I wasn't very clear, but I linked code examples in the issue above:
-  This is the example using A-Frame 0.9.2 https://codepen.io/navasmedrano/pen/RwWxLGV
- And, this is the example using A-Frame 0.8.0 https://codepen.io/navasmedrano/pen/mdepqXv

In both examples, I'm using the AR.js library without NFT support and using the Hiro marker.

Regarding the changes done to fix #60 as the cause of this behaviour... This is the example I mentioned before (https://codepen.io/navasmedrano/pen/xxwYGGv). In this example, I used A-Frame 0.8.0 and AR.js 2.1.8. If I remember correctly, click interaction worked properly both in mobile and desktop browsers some days ago. However, it's impossible that the changes done to fix #60 had affected somehow the 2.1.8 AR.js version hosted in jeromeetienne's old AR.js repository, right?
